driver/normalizer/tonode.go

```diff
@@ -0,0 +1,70 @@
+package normalizer
+
+import (
+	"strings"
+
+	"gopkg.in/bblfsh/sdk.v1/uast"
+)
+
+// ToNode is an instance of `uast.ObjectToNode`, defining how to transform an
+// into a UAST (`uast.Node`).
+//
+// https://godoc.org/gopkg.in/bblfsh/sdk.v1/uast#ObjectToNode
+var ToNode = &uast.ObjectToNode{
+	InternalTypeKey: "nodeType",
+	LineKey:         "attributes.startLine",
+	ColumnKey:       "attributes.startTokenPos",
+	OffsetKey:       "attributes.startFilePos",
+	EndLineKey:      "attributes.endLine",
+	EndColumnKey:    "attributes.endTokenPos",
+	EndOffsetKey:    "attributes.endFilePos",
+	TokenKeys: map[string]bool{
+		"name": true,
+		"text": true, // for comments
+	},
+	// PHP AST includes a map called attributes with several properties, should
+	// be ignored, otherwise fake nodes are created.
+	IsNode: func(v map[string]interface{}) bool {
+		_, ok := v["nodeType"]
+		return ok
+	},
+	// The parse returns multiple nodes instead of a single node, a fake node
+	// root node with the type "File" is created.
+	OnToNode: func(n interface{}) (interface{}, error) {
+		return map[string]interface{}{
+			"root": map[string]interface{}{
+				"nodeType": "File",
+				"children": n,
+			},
+		}, nil
+	},
+	Modifier: func(n map[string]interface{}) error {
+		// Sometimes the name if includes namespaces is given as an array in
+		// several parts. The parts are imploded into the name key.
+		if parts, ok := n["parts"].([]interface{}); ok {
+			n["name"] = sliceInterfaceToString(parts, "\\")
+			delete(n, "parts")
+		}
+
+		// In comments the positions doesn't follow the same schema of the
```

Comment:
* s/In comments the positions doesn't/Positions in comments don't/
* s/of/as/
* s/please/place/